index.rst

```diff
@@ -235,7 +235,9 @@ and the supported values associated with them:
      - Supported values
    * - ``indent_style``
      - Set to ``tab`` or ``space`` to use hard tabs or soft tabs respectively. The
-       values are case insensitive.
+       values are case insensitive. Note that this settings signifies the indentation
+       style to be **preferred whenever possible**. The exact semantics with examples can be
+       found in the indentation_ section
```

Comment:
How about changing the whole bullet point to:

```
Set to ``tab`` or ``space`` to use tabs or spaces for indentation, respectively. ``tab`` implies that an indentation is to be filled by as many hard tabs as possible, with the rest of the indentation filled by spaces. A non-normative explanation can be
found in the indentation_ section. The values are case insensitive.
```

Reply:
Sounds great, I'll just change the beginning of the seconds sentence to ``tab``.